tsdoc/src/details/StandardTags.ts

```diff
@@ -487,6 +487,42 @@ export class StandardTags {
     standardization: Standardization.Extended
   });
 
+  /**
+   * (TSDoc Built-in Directive)
+   */
+  public static readonly tsBuiltInJsx: TSDocTagDefinition = StandardTags._defineTag({
+    tagName: '@jsx',
+    syntaxKind: TSDocTagSyntaxKind.BlockTag,
+    standardization: Standardization.Extended
+  });
+
+  /**
+   * (TSDoc Built-in Directive)
+   */
```

Comment:
This comment doesn't give any information about the usage or intended meaning of the tag.


It should look something like this (hypothetical - actually I wasn't able to find the docs for these tags):

```ts
  /**
   * (Extended)
   *
   * This modifier instructs a JSX parser to treat the interface as a global attribute definition.
   *
   * @remarks
   * The TypeScript compiler recognizes this tag and specifies its behavior:
   *
   * @see {@link https://www.typescriptlang.org/docs/handbook/some-page.html#some-section|JSX directives}
   */
 ```